README.md

```diff
@@ -18,38 +21,48 @@ A simple application that loads the Safe Routes to School map that allows simple
 ## Updating the Backing Data
 
 ### To update the style
-- Login to [Mapbox Studio](https://studio.mapbox.com/) with the `safe-routes-to-school` username and password
+- Login to [Mapbox Studio](https://studio.mapbox.com/) with the
+  `safe-routes-to-school` username and password
 - Select the SRTS style
 - Update the style
-    * Do not update the name of the existing layers. This will break the toggles on the webpage
+    * Do not update the name of the existing layers. This will break the toggles
+      on the webpage
 - Hit the `Publish` button when done
 
 ### To update the map data
-The GeoJSON files from SFMTA/SFGIS were uploaded to Mapbox studio and converted into vector tilesets. These tilesets are referenced in the map style and updating them here wll update the displayed map.
 
-- Login to [Mapbox Studio](https://studio.mapbox.com/) with the `safe-routes-to-school` username and password
+The GeoJSON files from SFMTA/SFGIS were uploaded to Mapbox studio and converted
+into vector tilesets. These tilesets are referenced in the map style and
+updating them here wll update the displayed map.
+
+- Login to [Mapbox Studio](https://studio.mapbox.com/) with the
+  `safe-routes-to-school` username and password
 - Select `Tilesets`from the Top menu bar
 - Scroll to and select the tileset that you want to update
-- On the tileset page, click `Replace` on the right column and select the updated version of the GeoJSON file
+- On the tileset page, click `Replace` on the right column and select the
+  updated version of the GeoJSON file
 
-* If you are updating the school points data, also replace the `data/sfusd_school_pt.wgs84.json` file in this repo.
+The raw GeoJSON data going into these vector tilesets is in [`data/`](data/), as
+well as some notes on corrections and normalizations we made. Replacement
```

Comment:
The wording here is a bit confusing. Possible reword:
The map style depends on the GeoJSON properties, so the map style might break if the new data is missing any properties that exist in the old data.